lib/wait_for_socket.js

```diff
@@ -0,0 +1,30 @@
+var Promise = require('promise');
+var fs = require('fs');
+import Promise from 'Promise';
```

Comment:
Not sure how it worked for you, but I initially noticed it because it broke all the CI tests :)